kong/tools/time.lua

```diff
@@ -0,0 +1,102 @@
+local ffi = require "ffi"
+
+
+local C             = ffi.C
+local ffi_new       = ffi.new
```

Comment:
Since `ffi_new` is not in hot path, I think that we can remove this line and use `ffi.new` directly.

Reply:
removed